src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -49,14 +49,24 @@ public VisualEditorPageObject(WebDriver driver) {
   private List<WebElement> numList;
   @FindBy(css = "ul.ve-ce-branchNode > li")
   private List<WebElement> bullList;
-  @FindBy(css = ".ve-init-mw-viewPageTarget-surface")
-  private WebElement veEditorSurface;
+  @FindBy(css = ".oo-ui-popupToolGroup-handle")
+  private List<WebElement> toolsList;
+  @FindBy(css = ".oo-ui-tool-title")
+  private List<WebElement> insertMenuTools;
+  @FindBy(css = ".oo-ui-labelElement.oo-ui-optionWidget")
+  private List<WebElement> infoboxTemplatesList;
+  @FindBy(css = ".ve-ui-mwParameterPage-field .oo-ui-inputWidget textarea")
+  private List<WebElement> parametersFieldList;
+  @FindBy(css = ".oo-ui-buttonElement-button")
+  private List<WebElement> buttonsList;
   @FindBy(css = ".image.video.video-thumbnail.medium")
   private List<WebElement> videoNodes;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)